src/Steam.package/STDescriptorSystem.class/instance/initializeTable..st

```diff
@@ -0,0 +1,9 @@
+accessing
+initializeTable: aTable
+	| desbribedClass |
+
+	desbribedClass := aTable classModel describedClass.
+	desbribedClass basicNew magritteDescription childrenDo: [ :each |
+		aTable
+			createFieldNamed: each name
+			type: (platform text) ]
```

Comment:
We need to support more than just text ;)

Reply:
working on it